### PR TITLE
[IMP] Display More... in local language

### LINKED ISSUE #380: Display "More..." in local language
Currently, the option only allow to show the "More..." in English language.

### PR DESCRIPTION
**Pull request by S-mardii**
Fixed #380

### CODE REVIEW

File: widgets/query-resources-by-category-widget.php [hunk 1]
```diff
@@ -1,258 +1,293 @@
...
+    {
+        $title              = !empty($instance['title']) ? __(sanitize_text_field($instance['title']), 'wpckan') : __('Related resources by category', 'wpckan');
+        $type               = !empty($instance['type']) ? $instance['type'] : 'dataset';
+        $search_field       = !empty($instance['search_field']) ? sanitize_text_field($instance['search_field']) : 'taxonomy';
+        $organization       = isset($instance['organization']) ? $instance['organization'] : -1;
+        $organization_list  = [];
+        $sort               = isset($instance['sort']) ? $instance['sort'] : 'metadata_modified+desc';
+        $template           = isset($instance['template']) ? $instance['template'] : 'dataset-list';
+        $more_text          = !empty($instance['more_text']) ? $instance['more_text'] : 'More...';
+        $more_text_local    = !empty($instance['more_text_local']) ? $instance['more_text_local'] : 'បន្ថែម...';
```
Comment (EricSoroos) [inline widgets/query-resources-by-category-widget.php:156]: Do you want to use `_e()` for this default text?  It looks like that's our localization method for all the rest of this plugin.

Comment (S-mardii) [reply widgets/query-resources-by-category-widget.php:156]: Sorry that I missed seeing your suggestion. I will check and update the code. Thank you.

**Comment by S-mardii** [comment]
@EricSoroos I made the code change as you suggested.

**Comment by S-mardii** [comment]
@EricSoroos please help review this PR again. Thank you

**Comment by S-mardii** [comment]
@EricSoroos any update on this?

**Pull request merged**